src/Sleep.js

```diff
@@ -1,58 +1,40 @@
-class Sleep {
-  constructor(sleepData, userID) {
-    this.sleepData = sleepData;
-    this.userID = userID;
-  }
-
-  findUser() {
-    return this.sleepData.filter(user => user.userID === this.userID);
-  }
-
-  returnWeekOfData(week, userData) {
-    return [...userData].splice((-7 * week), 7);
-  }
+import UserFitness from "./UserFitness";
 
-  returnWeek(week) {
-    var specificUser = this.findUser()
-    return [...specificUser].splice(-7 * week, 7).map(day => day.date);
+class Sleep extends UserFitness {
+  constructor(sleepData, user) {
+    super(sleepData, user);
   }
```

Comment:
This is good stuff. Each sub-class gets only the info it needs from the parent class.